### ISSUE TITLE
RMSD calculation of two RNA residues returns nan

### ISSUE THREAD

**Issue by krother** [rasbt/biopandas#33]
The following code sniplet returns a nan, although the two residues have the same size and order of atom names:

`from biopandas.pdb import PandasPdb

ehz = PandasPdb().fetch_pdb('1ehz')

at = ehz.df['ATOM']

a64 = at[at['residue_number'] == 64]
a66 = at[at['residue_number'] == 66]

r = PandasPdb.rmsd(a64, a66)
print(r)
`

**Comment by rasbt** [comment]
Thanks for the note! To be honest, I can't explain this issue, but I believe this can be easily fixed (working on it right now, and I will add this case to the unit tests)

A little bit more about this issue...
Following up on your code, I get the following, which is expected:

    >>> a64['x_coord'].dtype
    dtype('float64')
    >>> a66['x_coord'].dtype
    dtype('float64')

Also, the 2 series look normal to me upon inspection:

```python
>>> a64['x_coord']
1053    63.129
1054    62.528
1055    63.807
1056    62.008
1057    61.194
1058    60.204
1059    60.917
1060    59.459
1061    58.374
1062    59.036
1063    57.994
1064    60.283
1065    61.176
1066    62.220
1067    62.766
1068    62.051
1069    62.152
1070    63.082
1071    61.261
1072    60.346
1073    60.169
1074    61.070
Name: x_coord, dtype: float64
```

and 

```python
>>> a66['x_coord']
1098    54.074
1099    52.701
1100    55.214
1101    54.173
1102    53.339
1103    53.671
1104    55.037
1105    53.666
1106    52.326
1107    54.535
1108    53.773
1109    55.595
1110    56.838
1111    57.216
1112    58.365
1113    58.784
1114    59.943
1115    60.924
1116    60.042
1117    59.056
1118    57.922
1119    57.850
Name: x_coord, dtype: float64
```

However, when I try to subtract these 2 arrays, I get the following:

```python
>>> a64['x_coord'] - a66['x_coord']
1053   NaN
1054   NaN
1055   NaN
1056   NaN
```

I believe this is due to the subtraction being performed on (non) matching pd.Series indices. 

Since the following works:

```python
>>> import pandas as pd
>>> pd.Series([1., 2., 3.]) - pd.Series([1., 2., 5.])
0    0.0
1    0.0
2   -2.0
dtype: float64
```

but the following doesn't:

>>> pd.Series([1., 2., 3.]) - pd.Series([1., 2., 5.], index=[4, 5, 7])

I never paid attention to that in this context since the molecules I compared usually had the same shape (or matching atom indices). That's a big gotcha. Thanks for pointing that out. However, using NumPy arrays instead of series should solve the issue. Will fix that in a moment and send you a note once that's resolved.

**Comment by rasbt** [comment]
should be fixed now via #34
